styles/layout.less

```diff
@@ -117,6 +117,18 @@ span.button {
     padding: 1px 4px;
 }
 
+.icon_button {
+    font-size: 60%;
+    border: none;
+    background: none;
+    cursor: pointer;
+    color: @link-color;
+}
+
+.icon_button:hover {
```

Comment:
I kinda like this but also wonder if it would just be a snowflake and that it creates an implicit dependency for a theme between the link color-and the table-cell-base-lowc color.

@chrismiceli / @srjfoo - what do y'all think?

Reply:
I guess it depends on the themes. Does this work in all themes for good contrast and is it limiting us? I could see maybe splitting it up in a future theme? Maybe for now it is ok?

However, looking throughout the site, it is weird. There isn't much hover so it feels strange that we need something new for this. Would it be better to make it a normal button like most of the other icon buttons?
![image](https://github.com/user-attachments/assets/7a1c85c0-ae49-49b9-9fd5-cdf644c0fc21)

Reply:
> Would it be better to make it a normal button like most of the other icon buttons?

I think the only reason it needs all this complexity is because we want it to POST instead of GET so the "bookmark this page" URL isn't bookmarkable -- if we could use `<a>` to do a POST not a GET we wouldn't need the JS stuff at all. But alas, here we are. Feels like making this act like just another link is _probably_ the most intuitive thing UI-wise?

Reply:
It seems to me that the anchor element is really intended to navigate to a new page and its right-click menu has  items "Open link in new tab" etc. If this is not meaningful in the context then the element should not be an anchor but a button of some sort. But we can make it have any appearance we like: a button with text or with an icon, or just an icon or a piece of text with some distinctive style. And we can show that is is clickable by changing the cursor or the appearance of the 'button' or both.

Reply:
> I kinda like this but also wonder if it would just be a snowflake and that it creates an implicit dependency for a theme between the link color-and the table-cell-base-lowc color.
> 
> @chrismiceli / @srjfoo - what do y'all think?

Works for me!